src/server/game/DataStores/DB2Stores.h

```diff
@@ -258,7 +267,10 @@ class TC_GAME_API DB2Manager
     DB2StorageBase const* GetStorage(uint32 type) const;
 
     void LoadHotfixData();
+    void LoadHotfixBlob();
     std::map<uint64, int32> const& GetHotfixData() const;
+    std::multimap<uint32 /*hash*/, HotfixBlobData> const& GetHotfixBlob();
```

Comment:
Another question... I need clariification: One hash has multiple HotfixBlobData? Is that the reason why you choose a multimap?